lib/ddtrace/contrib/http/instrumentation.rb

```diff
@@ -63,10 +66,9 @@ def request(req, body = nil, &block) # :yield: +response+
             end
           end
 
-          def annotate_span!(span, request, response)
+          def annotate_span_with_req!(span, request)
```

Comment:
It might be easier to read just to expand these names to `annotate_span_with_request!` and `annotate_span_with_response!` respectively. Longer names like this are okay as long as they're descriptive and concise as possible.

Reply:
yup makes sense, updated.